### ISSUE TITLE
nflow errors

### ISSUE THREAD

**Issue by amylin105** [noegroup/bgflow#78]
Hello. I have followed all the required installation instructions and I'm testing it out. I have a pdb file and dcd trajectory inputted. It runs through the allotted time I give it on the cluster. However, nothing spits out. There are two errors in the output file, both related to nflows:

`ERROR: Could not find a version that satisfies the requirement nflows (from versions: none)`
`ERROR: No matching distribution found for nflows`

I was wondering if you could provide any insight on that.

Thank you in advance.

**Comment by LeonKlein** [comment]
Hi, is the package `nflows` installed? You can install it with `pip install nflows`.

**Comment by amylin105** [comment]
Yes, nflows is installed

**Comment by LeonKlein** [comment]
Okay, can you give some more information what you are doing?

**Comment by amylin105** [comment]
Of course. We're trying to see if we can train the BG to predict the 3D structures of strands of DNA. We input a pdb and a dcd trajectory file to provide info on the topology and the dynamics of the DNA, respectively. Then, we specify the training parameters. The goal right now is just to get the energies after each training step. What we've noticed today is that it continues to run despite the error message and actually does give us the energies. The follow-up question to this would be does this error affect any of the training. 

In the submission file, we pip install all the required packages and their dependencies and then run. The full error message is as follows: 
`ERROR: Could not find a version that satisfies the requirement nflows (from versions: none)
ERROR: No matching distribution found for nflows`

Thank you!

**Comment by Olllom** [comment]
This error does not seem to be related to bgflow. Please check which command is causing the error. Make sure that you are not calling the system "pip" but the one from the conda environment/virtualenv that you want to use.
